src/Raven.Server/Smuggler/Documents/DatabaseDestination.cs

```diff
@@ -2192,16 +2192,22 @@ public async ValueTask DisposeAsync()
                 await SendCommandsAsync();
             }
 
-            public async ValueTask WriteSubscriptionAsync(SubscriptionState subscriptionState)
+            public async ValueTask WriteSubscriptionAsync(SubscriptionState subscriptionState, bool includeState = false)
```

Comment:
Create a dedicated method which will be used only by the action `WriteSubscriptionWithStateAsync`

Reply:
This method then will create a command which will be passed to some WriteSubscriptionInternalAsync method that will be invoked by both of them

Reply:
Done